cluster-up/cluster/k8s-provider-common.sh

```diff
@@ -4,6 +4,19 @@ set -e
 
 source ${KUBEVIRTCI_PATH}/cluster/ephemeral-provider-common.sh
 
+function _label_workers() {
+    # Label all the non master nodes as workers, we have to do here since, after k8s 1.16 is not possible to do
+    # at kubelet [1]
+    # [1] https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/user/troubleshooting.md
+    number_of_nodes=$($kubectl get node  --no-headers |wc -l)
```

Comment:
in rare case that kubectl temporally fails this would give 0 because the wc at the end
so maybe worth to first make sure the left most command success, 
theoretically set -o pipefail && $kubectl might work , but one time tried it and it didnt , guess i didnt do it right,
just until can be used if command splitted

Reply:
Ahh thanks, let me fix it.

Reply:
In case something bad happaned during deployment and you reach to this point,
`kubectl get nodes` will fail with server not reachable error, then wc -l will return 0.
I think we should replace else with` elif [ $number_of_nodes -gt 1 ]`

We can also verify that all nodes that the user requested are up with comparing $number_of_nodes with KUBEVIRT_NUM_NODE

Reply:
the pipe should fail it as a whole
the only other alternative i see is to do a redo on the command
else the value that will be used isnt right